<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PROGERSS</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QTreeWidget" name="treeWidget">
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <column>
       <property name="text">
        <string>Предметная область</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <item>
       <property name="text">
        <string>База данных</string>
       </property>
       <property name="icon">
        <iconset resource="img.qrc">
         <normaloff>:/img/database.png</normaloff>:/img/database.png</iconset>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEnabled</set>
       </property>
       <item>
        <property name="text">
         <string>Поставщики</string>
        </property>
        <property name="icon">
         <iconset resource="img.qrc">
          <normaloff>:/img/Table.png</normaloff>:/img/Table.png</iconset>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEnabled</set>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Аппаратура</string>
        </property>
        <property name="icon">
         <iconset resource="img.qrc">
          <normaloff>:/img/Table.png</normaloff>:/img/Table.png</iconset>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEnabled</set>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Продажи</string>
        </property>
        <property name="icon">
         <iconset resource="img.qrc">
          <normaloff>:/img/Table.png</normaloff>:/img/Table.png</iconset>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEnabled</set>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Справочники</string>
        </property>
        <property name="icon">
         <iconset resource="img.qrc">
          <normaloff>:/img/HelpBook.png</normaloff>:/img/HelpBook.png</iconset>
        </property>
        <item>
         <property name="text">
          <string>Сотрудники</string>
         </property>
         <property name="icon">
          <iconset resource="img.qrc">
           <normaloff>:/img/profile.png</normaloff>:/img/profile.png</iconset>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsEnabled</set>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Форма оплаты</string>
         </property>
         <property name="icon">
          <iconset resource="img.qrc">
           <normaloff>:/img/coins.png</normaloff>:/img/coins.png</iconset>
         </property>
        </item>
       </item>
      </item>
      <item>
       <property name="text">
        <string>Запросы к данным</string>
       </property>
       <property name="icon">
        <iconset resource="img.qrc">
         <normaloff>:/img/Querys.png</normaloff>:/img/Querys.png</iconset>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEnabled</set>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Отчеты</string>
       </property>
       <property name="icon">
        <iconset resource="img.qrc">
         <normaloff>:/img/Reports.png</normaloff>:/img/Reports.png</iconset>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEnabled</set>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="page0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>45</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Для работы выберите объект базы данных 
 в предметной области</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignHCenter|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="img.qrc">:/img/database.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page1">
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0" colspan="4">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>358</width>
              <height>18</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Область представления данных</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0" colspan="5">
         <widget class="QTableView" name="tableView">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QWidget" name="panel" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>3</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>3</number>
           </property>
           <item>
            <widget class="QPushButton" name="pbAdd">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="img.qrc">
               <normaloff>:/img/addrow.png</normaloff>:/img/addrow.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbEdit">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="img.qrc">
               <normaloff>:/img/editrow.png</normaloff>:/img/editrow.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbDel">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="img.qrc">
               <normaloff>:/img/deleterow.png</normaloff>:/img/deleterow.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbRefresh">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="img.qrc">
               <normaloff>:/img/update.png</normaloff>:/img/update.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="1">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>202</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="leFilter">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>270</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="pbFilter">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>21</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="img.qrc">
            <normaloff>:/img/filter.png</normaloff>:/img/filter.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QPushButton" name="pbClear">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>21</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="img.qrc">
            <normaloff>:/img/clear.png</normaloff>:/img/clear.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page2">
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>45</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Для ввода или редактирования
 выберите справочник</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignHCenter|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="img.qrc">:/img/HelpBook.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page3">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Запросы к базе данных</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QListWidget" name="listQuerys">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="iconSize">
           <size>
            <width>35</width>
            <height>35</height>
           </size>
          </property>
          <property name="flow">
           <enum>QListView::TopToBottom</enum>
          </property>
          <property name="isWrapping" stdset="0">
           <bool>false</bool>
          </property>
          <property name="resizeMode">
           <enum>QListView::Fixed</enum>
          </property>
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="batchSize">
           <number>100</number>
          </property>
          <item>
           <property name="text">
            <string>Последняя продажа</string>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="icon">
            <iconset resource="img.qrc">
             <normaloff>:/img/query.png</normaloff>:/img/query.png</iconset>
           </property>
           <property name="flags">
            <set>ItemIsSelectable|ItemIsEnabled</set>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Отобразить продажи за указанный день</string>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="icon">
            <iconset resource="img.qrc">
             <normaloff>:/img/query.png</normaloff>:/img/query.png</iconset>
           </property>
           <property name="flags">
            <set>ItemIsSelectable|ItemIsEnabled</set>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Отобразить поставки за указанный день</string>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="icon">
            <iconset resource="img.qrc">
             <normaloff>:/img/query.png</normaloff>:/img/query.png</iconset>
           </property>
           <property name="flags">
            <set>ItemIsSelectable|ItemIsEnabled</set>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Подсчет суммы продаж за период</string>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="icon">
            <iconset resource="img.qrc">
             <normaloff>:/img/query.png</normaloff>:/img/query.png</iconset>
           </property>
           <property name="flags">
            <set>ItemIsSelectable|ItemIsEnabled</set>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Товары в наличии</string>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="foreground">
            <brush brushstyle="NoBrush">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </property>
           <property name="icon">
            <iconset resource="img.qrc">
             <normaloff>:/img/query.png</normaloff>:/img/query.png</iconset>
           </property>
           <property name="flags">
            <set>ItemIsSelectable|ItemIsEnabled</set>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page4">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Отчеты</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QListWidget" name="listReports">
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="iconSize">
           <size>
            <width>35</width>
            <height>35</height>
           </size>
          </property>
          <property name="spacing">
           <number>5</number>
          </property>
          <item>
           <property name="text">
            <string>Список сотрудников</string>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="icon">
            <iconset resource="img.qrc">
             <normaloff>:/img/report.png</normaloff>:/img/report.png</iconset>
           </property>
           <property name="flags">
            <set>ItemIsSelectable|ItemIsEnabled</set>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Прайс-лист</string>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="icon">
            <iconset resource="img.qrc">
             <normaloff>:/img/report.png</normaloff>:/img/report.png</iconset>
           </property>
           <property name="flags">
            <set>ItemIsSelectable|ItemIsEnabled</set>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Список поставщиков</string>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="icon">
            <iconset resource="img.qrc">
             <normaloff>:/img/report.png</normaloff>:/img/report.png</iconset>
           </property>
           <property name="flags">
            <set>ItemIsSelectable|ItemIsEnabled</set>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Количество продаж за период</string>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="icon">
            <iconset resource="img.qrc">
             <normaloff>:/img/report.png</normaloff>:/img/report.png</iconset>
           </property>
           <property name="flags">
            <set>ItemIsSelectable|ItemIsEnabled</set>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Количество поставок за период</string>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="icon">
            <iconset resource="img.qrc">
             <normaloff>:/img/report.png</normaloff>:/img/report.png</iconset>
           </property>
           <property name="flags">
            <set>ItemIsSelectable|ItemIsEnabled</set>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>862</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
